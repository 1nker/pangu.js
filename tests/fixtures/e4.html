<div id="e4">
    <p>123漢字abc</p>
    <textarea>中文abc漢字5566</textarea>
    <p>中文7788</p>
    <textarea><div>中文abc漢字5566<span>漢字123</span></div></textarea>
    <label>選項(必填)</label>
    <input type="text" value="中文123漢字abc" />
</div>
