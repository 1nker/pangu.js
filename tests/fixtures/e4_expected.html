<div id="e4">
    <p>123 漢字 abc</p>
    <textarea>中文abc漢字5566</textarea>
    <p>中文 7788</p>
    <textarea><div>中文abc漢字5566<span>漢字123</span></div></textarea>
    <label> 選項 (必填)</label>
    <input type="text" value="中文123漢字abc" />
</div>
